api/v1/schemas/auth.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel, EmailStr
+from typing import Dict
+from datetime import datetime
+import uuid
+
+
+class UserCreate(BaseModel):
+    username: str
+    email: str
+    password: str
+    first_name: str | None = None
+    last_name: str | None = None
+
```

Comment:
`passowrd` is not being properly validated

Reply:
> `passowrd` is not being properly validated

i have fixed this